packages/js/src/utils/errors.ts

```diff
@@ -7,6 +7,14 @@ const logtoErrorCodes = Object.freeze({
   },
 });
 
+const callbackUriErrorCodes = Object.freeze({
+  redirectUriMismatched: 'Redirect URI mismatched',
+  errorFound: 'Error found',
+  missingState: 'Missing state',
+  stateMismatched: 'State mismatched',
+  missingCode: 'Missing code',
```

Comment:
would you mind following our JS error code and phrase convention, i.e. use `pascal_case` for the code (key)?

Reply:
Ok. I will do it later. LOG-1416

Reply:
Correction: should `snake_case`